sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -200,6 +203,7 @@ private static void internalLogin(@NonNull final String jwt, final KinCallback<V
 			sendLoginFailed(exception, loginCallback);
 		}
 
+		eventLogger.send(UserLogoutRequested.create());
```

Comment:
I suggest changing isSameUser to something like getLoginStatus(jwt) which returns  IntDef of logut|sameUser|diffrentUser